### ISSUE TITLE
Hasura actions template

### ISSUE THREAD

**Issue by tiagob** [tiagob/create-full-stack#53]

**Comment by austinrivas** [comment]
Since you are already providing the pulumi aws provider I think the best way to do this is by provisioning an aws lambda function and api gateway endpoint that is used by a hasura action.

Let me know if you need a hand with this, I have some generic hasura / lambda boilerplate that might be helpful.

**Comment by tiagob** [comment]
Agreed. I like that. I can handle this. What’s a simple example for an
action with a Todo app?

**Comment by austinrivas** [comment]
The easiest action I could think of would be to just provide an echo function that sends back whatever you send it, but if you want something specific to the todo app maybe an email notification on item completion?

I've written some code to extract and validate hasura action body and headers using the codegen types, but I think it might be a little to abstract for a simple example.
